Brewfile

```diff
@@ -0,0 +1,25 @@
+tap "dpogue/plasma-deps"
+
+brew "asio"
+brew "cairo"
+brew "cmake"
+brew "curl"
+brew "expat"
+brew "freetype"
+brew "libjpeg-turbo"
+brew "libogg"
+brew "libpng"
+brew "libtool"
+brew "libvorbis"
+brew "libvpx"
+brew "libwebm"
+brew "openal-soft"
+brew "openssl"
+brew "opus"
+brew "physx"
+brew "python@3.9"
```

Comment:
Do we have to hardcode the minor version? The Windows build is now on 3.10...

Reply:
It looks like `python@3.x` might work, I'll give it a shot

Reply:
`python@3.x` does not work, but I'm updating it to `python@3.10` now 😞